packages/blueprints-integration/src/context.ts

```diff
@@ -128,6 +128,13 @@ export interface IActionExecutionContext extends IShowStyleUserContext, IEventCo
 			pieceMetaDataFilter?: any // Mongo query against properties inside of piece.metaData
 		}
 	): IBlueprintPieceInstance | undefined
+	findLastScriptedPieceOnLayer(
+		sourceLayerId: string,
```

Comment:
is there a reason that `sourceLayerId` doesnt allow an array?

A documentation comment would be good for the method too

Reply:
I've changed it to accept an array